example_simple_classification.py

```diff
@@ -41,7 +41,7 @@ def main():
     # Run 4 built in algorithms to see which one looks best at first
     hcai.ensemble()
 
-    hcai.get_advanced_features().advanced_random_forest_classifier()
+    hcai.get_advanced_features().random_forest_classifier()
 
```

Comment:
1. Get advanced is a way to get the underlying instance of the advanced class if you need one or two advanced features without the hassle of using the advanced class directly. I don't anticipate widespread use, and the implementation is trivial.

2. Adding new algorithms is no longer an issue. There is no need to push back on new algorithms, especially if they are only available in the advanced class first.

3. Thanks! Making things better is my jam.